pkg/services/secrets/kvstore/fallback.go

```diff
@@ -0,0 +1,67 @@
+package kvstore
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+)
+
+type KVStoreWithFallback struct {
+	log      log.Logger
+	store    SecretsKVStore
+	fallback SecretsKVStore
+}
+
+func NewKVStoreWithFallback(store SecretsKVStore, fallback SecretsKVStore) *KVStoreWithFallback {
+	return &KVStoreWithFallback{
+		log:      log.New("secrets.kvstore"),
```

Comment:
pass existing logger into here maybe?

Reply:
Hmm all the other wrappers call log.New() not sure if we should change that now or leave that for a future refactor.

Reply:
Okay, yeah it's good as-is then 👍

Reply:
Left like this for now to follow the same pattern, this could be changed on a future refactor.